group_members.go

```diff
@@ -151,6 +151,55 @@ func (s *GroupMembersService) GetGroupMember(gid interface{}, user int, options
 	return gm, resp, err
 }
 
+// BillableGroupMember represents a GitLab billable group member.
+//
+// GitLab API docs: https://docs.gitlab.com/ee/api/members.html#list-all-billable-members-of-a-group
+type BillableGroupMember struct {
+	ID             int     `json:"id"`
+	Username       string  `json:"username"`
+	Name           string  `json:"name"`
+	State          string  `json:"state"`
+	AvatarURL      string  `json:"avatar_url"`
+	WebURL         string  `json:"web_url"`
+	LastActivityOn ISOTime `json:"last_activity_on"`
+}
```

Comment:
It seems you missed the `email` field (in between the `WebURL` and the `LastActivityOn` fields).

Additionally I wonder if we shouldn't just add the missing fields to the `GroupMember` struct and use that one instead. What do you think?

Reply:
See you already addressed this in your comments 😊 I'm good with using this new struct if you add the missing `email` field...

Reply:
> It seems you missed the email field (in between the WebURL and the LastActivityOn fields).

🤦 The email field wasn't in the first example billable group member object in the list so I missed it. Good catch!

Reply:
Fixed. Thanks @svanharmelen !